submit/submit.sh

```diff
@@ -3,11 +3,22 @@
 export SPARK_MASTER_URL=spark://${SPARK_MASTER_NAME}:${SPARK_MASTER_PORT}
 export SPARK_HOME=/spark
 
+function is_uri() {
+    regex='(https?|hdfs|file)://[-A-Za-z0-9\+&@#/%?=~_|!:,.;]*[-A-Za-z0-9\+&@#/%=~_|]'
+    if [[ $1 =~ $regex ]]
+    then 
+        echo "true"
+    else
+        echo "false"
+    fi
+}
+
+
 /wait-for-step.sh
 
 
 /execute-step.sh
-if [ -f "${SPARK_APPLICATION_JAR_LOCATION}" ]; then
+if [[ `is_uri $SPARK_APPLICATION_JAR_LOCATION`=="true" || -f "${SPARK_APPLICATION_JAR_LOCATION}" ]]; then
```

Comment:
I'm afraid that by checking only `is_uri` we exclude the possibility to add this variable via [bde-spark-template*](https://github.com/big-data-europe/docker-spark/tree/master/template) images. 
Let me test it first if the template overrides this variable by passing it on the template image since it uses spark-submit as a base image.

Reply:
Is there a test that I can run to verify it?

Reply:
Hi @ColonelLanda ,
there are some examples that we provide on BDE or [SANSA](https://github.com/SANSA-Stack) but for simplicity, I just created a [spark-starter](https://github.com/GezimSejdiu/spark-starter) example for testing this. Give it a try and let me know if you need more help on running the example over there.
Best regards,